worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -86,19 +90,40 @@ private[celeborn] class Worker(
     conf,
     Math.min(64, Math.max(4, Runtime.getRuntime.availableProcessors())))
 
```

Comment:
Are we going to support for disabling the existing behavior via a config in a future pr ? (`workerArgs.port` - which is not secure and is used for both s2s and c2s connections)

Reply:
Yes, will be making that change in a future pr. I haven't created a jira yet.

Reply:
Created https://issues.apache.org/jira/browse/CELEBORN-1268 to address this.